packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Deployment/Netlify.js

```diff
@@ -37,6 +41,17 @@ class NetlifyDeployment extends Component {
 
     const template = getTemplate(editor.currentSandbox.template);
     const { show } = this.state;
+    const functionDirectory = getNetlifyConfig(editor.currentSandbox).functions
+      ? resolveDirectory(
```

Comment:
Keep in mind that this function throws if it doesn't find anything.

Reply:
yap! That's why I only run it if there are functions and toml file :)

Reply:
I'm mostly curious what happens when the configuration links to a directory that doesn't exist. Not sure if there's something for that? 🤔

Reply:
I will try that!

Reply:
It did break :(

Added a try catch, if it can't resolve just returns an empty array